dv/uvm/core_ibex/sim_makefrag_gen.py

```diff
@@ -79,7 +94,7 @@ def main():
         'Generates a makefile fragment for use with the Ibex DV makefile that '
         'sets up sim specific variables'))
 
-    sim_fns = {'vcs': gen_vcs_makefrag, 'riviera': gen_riviera_makefrag, 'xlm': gen_xlm_makefrag}
+    sim_fns = {'vcs': gen_vcs_makefrag, 'riviera': gen_riviera_makefrag, 'xlm': gen_xlm_makefrag, 'questa': gen_questa_makefrag}
```

Comment:
One minor thing: We aim to keep line sizes short in python (79 characters or less as specified by PEP8 https://www.python.org/dev/peps/pep-0008/#maximum-line-length) so could you break this line up?

You weren't to know as we don't actually document our preferred Python style in the Ibex repository as far as I can tell! The OpenTitan documentation has the full details: https://docs.opentitan.org/doc/rm/python_coding_style/

Reply:
will break it up, one per line

Reply:
it's updated accordingly